getting_started/first_steps.md

```diff
@@ -14,26 +14,51 @@ Deno is a runtime for JavaScript/TypeScript which tries to be web compatible and
 use modern features wherever possible.
 
 Browser compatibility means a `Hello World` program in Deno is the same as the
-one you can run in the browser:
+one you can run in the browser.
+
+Create file locally called `first_steps.ts` and copy and paste the code line
+below:
 
 ```ts
 console.log("Welcome to Deno!");
 ```
 
-Try the program:
+## Running Deno Programs
+
+Now to run the program from the terminal:
+
+```shell
+deno run first_steps.ts
+```
+
+Deno also has the ability to execute scripts from URLs. Deno
+[hosts a library](https://deno.land/std@0.103.0/examples) of example code, one
+of which is a `Hello World` program. To run that hosted code, do:
 
 ```shell
-deno run https://deno.land/std@$STD_VERSION/examples/welcome.ts
+deno run https://deno.land/std@0.103.0/examples/welcome.ts
 ```
 
+## $STD_VERSION
```

Comment:
I am not sure why this was marked as resolved.  The section doesn't make any sense really, because this is what gets rendered:

![First_Steps___Manual___Deno](https://user-images.githubusercontent.com/1282577/196598899-b89c542a-e8df-4ddd-aca2-857bfa427863.png)

So I am not sure we need to even mention it.  It only would need to be mentioned in the README for contributions.

Reply:
How do you escape the variable?